content/v2.0/influxdb-templates/stacks/_index.md

```diff
@@ -39,52 +37,118 @@ Use stacks to save time in the following cases:
-   deploy the changes to your instance. Note the final line in this script `./install.sh prod` passes in a the `prod` configuration variable,
-   which grabs the INFLUX_TOKEN, INFLUX_ORG, and INFLUX_URL to install everything into the Cloud instance.
-8. Click **Actions** in the repository to see all the tasks in the `deploy-to-cloud` workflow.
-
-   Optionally, you can set up another continuous integration platform, such as [Circle CI](https://circleci.com/).
-
-#### Make a change in your repository
-
-Once you've set up your GitHub repository to back your InfluxDB instance, follow the steps below
-to make a change to your new repository and automatically deploy the change to your InfluxDB instances.
-
-1. In your integrated development environment (IDE), open the repository, make a change locally, and then test against your InfluxDB instance.
-2. Commit your change. For more information, see [GitHub docs about commits](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/committing-and-reviewing-changes-to-your-project#about-commits).
-3. Create a pull request. For more information, see [GitHub docs for creating a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
-4. Merge your pull request. For more information, see [GitHub docs for merging a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/merging-a-pull-request). Your change is automatically deployed to your InfluxDB instance.
+As long as your deployment automation software allows you to run a custom script, you can set it up to run the same script you've built locally, but against another environment. Here is an example of using a custom Github Action to do this:
+
+```yml
+name: deploy-influxdb-resources
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.ref }}
+    - name: Deploys repo to cloud
+      env:
+        # These secrets can be configured in the Github repo to connect to 
+        # your InfluxDB instance.
+        INFLUX_TOKEN: ${{ secrets.INFLUX_TOKEN }}
+        INFLUX_ORG: ${{ secrets.INFLUX_ORG }}
+        INFLUX_URL: ${{ secrets.INFLUX_URL }}
+        GITHUB_REPO: ${{ github.repository }}
+        GITHUB_BRANCH: ${{ github.ref }}  
+      run: |
+        cd /tmp
+        wget https://dl.influxdata.com/platform/nightlies/influx_nightly_linux_amd64.tar.gz
+        tar xvfz influx_nightly_linux_amd64.tar.gz
+        sudo cp influx_nightly_linux_amd64/influx /usr/local/bin/
+        cd $GITHUB_WORKSPACE
+        # This runs the script to set up your stacks
+        chmod +x ./setup.sh
+        ./setup.sh prod
+```
+
+Check out the complete [Github Actions documentation](https://github.com/features/actions) for more information about using these in your project.
```

Comment:
For more info..., check out [Github Actions documentation](https://github.com/features/actions).